docs/articles/documentation/using-testcafe/programming-interface/Runner.md

```diff
@@ -0,0 +1,258 @@
+
+You are free to mix different types of objects in one function call. The `browsers` function concatenates the settings when called several times.
+
+#### Using browser aliases
+
+```js
+runner.browsers(['safari', 'chrome']);
+```
+
+#### Specifying the path to the browser executable
+
+```js
+runner.browsers('C:\\Program Files\\Internet Explorer\\iexplore.exe');
+```
+
+#### Specifying the path with command line parameters
+
+```js
+runner.browsers({
+    path: 'C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe',
+    cmd: '--new-window'
+});
+```
+
+#### Passing the remote browser connection
+
+```js
+const createTestCafe   = require('testcafe');
+const testCafe         = await createTestCafe('localhost', 1337, 1338);
+
+const remoteConnection = testcafe.createBrowserConnection();
+
+// Outputs remoteConnection.url to visit it from the remote browser.
+console.log(remoteConnection.url);
+
+remoteConnection.once('ready', async () => {
+    await testCafe
+        .createRunner()
+        .browsers(remoteConnection)
+        .run();
+});
+```
+
+### <a class="anchor" name="screenshots"></a>screenshots
+
+Enables the test runner to take screenshots of the tested webpages.
+
+```text
+screenshots(path [, takeOnFails]) → this
+```
+
+Parameter                  | Type    | Description                                                                   | Default
+-------------------------- | ------- | ----------------------------------------------------------------------------- | -------
+`path`                     | String  | The path to which the screenshots will be saved.
+`takeOnFails` *(optional)* | Boolean | Specifies if screenshots should be taken automatically whenever a test fails. | `false`
+
+The `screenshots` function must be called in order to enable TestCafe to take screenshots whenever the `screenshot` action is called from test code.
+If the `screenshots` function is not called, the `screenshot` action is ignored.
+
+The `takeOnFails` parameter handles a separate scenario of capturing the webpage. Set it to `true` to make TestCafe take a screenshot whenever a test fails.
+
+**Example**
+
+```js
+runner.screenshots('/home/user/screenshots/', true);
+```
+
+### <a class="anchor" name="reporter"></a>reporter
+
+Configures the test runner's reporting feature.
+
+```text
+reporter(name [, outStream]) → this
+```
+
+Parameter                | Type                        | Description                                     | Default
+------------------------ | --------------------------- | ----------------------------------------------- | --------
+`name`                   | String                      | The name of the reporter to use.
+`outStream` *(optional)* | Writable Stream implementer | The stream to which the report will be written. | `stdout`
+
+Reporters are plugins used to output test run reports in a certain format.
```

Comment:
> Since we mention reporters here and in the CLI topic, are we going to create a separate section about them (like Aliases)?

It's don't make much sense since: a)unlike aliases, reporter is a simple and basic concept that doesn't require any additional explanation b)we have link to the officially supported reporters there we have visual explanation in form of screenshots